### PR TITLE
Changing names of functions

### PR DESCRIPTION
**Pull request by Ethan-Master-Coding**
Changed the name of `add()` to `sum()`

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/javascript/concepts/functions/functions.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Function declarations are used to create named functions. These functions can be
 The example code provided contains a function named `add()` that takes in 2 values and prints the sum of those numbers:
 
 ```js
-function add(number1, number2) {
+function sum(number1, number2) {
```
Comment (yangc95) [inline content/javascript/concepts/functions/functions.md:30]: Let's adjust the explanations above to match our change here 👍

**Review by yangc95 — COMMENTED** [review]
Finished with the first review. Please tag me once you've made the appropriate changes!

**Review by Ethan-Master-Coding — COMMENTED** [review]
All edits look good and are correct.

**Review by yangc95 — APPROVED** [review]
Looks ready to be merged 👍

**Review by Ethan-Master-Coding — COMMENTED** [review]
All edits look good and are correct.

File: content/javascript/concepts/functions/functions.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ Function declarations are used to create named functions. These functions can be
 - An optional list of parameters separated by commas enclosed by a set of parentheses `()`.
 - A function body enclosed in a set of curly braces `{}`.
 
-The example code provided contains a function named `add()` that takes in 2 values and prints the sum of those numbers:
+The example code provided contains a function named `sum()` that takes in 2 values and prints the sum of those numbers:
```
Comment (Dusch4593) [inline content/javascript/concepts/functions/functions.md:27]:
```suggestion
The example code provided contains a function named `sum()` that takes in two values and prints their sum:
```

Comment (Dusch4593) [reply content/javascript/concepts/functions/functions.md:27]: Recommending this small change and we'll be good to merge! 👍

**Comment by Ethan-Master-Coding** [comment]
@Dusch4593 accepted the suggestions.

**Review by Dusch4593 — APPROVED** [review]
LGTM! I made a small change above @Ethan-Master-Coding's changes (in the list) by surrounding "function" with back ticks in "The function keyword".

**Comment by Dusch4593** [comment]
@Ethan-Master-Coding your changes to the JavaScript Functions entry are now live! 🎉 

https://www.codecademy.com/resources/docs/javascript/functions

**Pull request merged**